### ISSUE TITLE
Bots only join Spectators.

### ISSUE THREAD

**Issue by PRinceBhai** [EfeDursun125/CS-EBOT#5]
Even with ebot_add_ct or ebot_add_t Bots only join Spectator, I haven't even touched the config. How do I resolve this?

Metamod p 1.21
reGameDLL 5.21.0.566

**Comment by EfeDursun125** [comment]
Do you have rehlds? use any auto team join plugin to fix this

**Comment by PRinceBhai** [comment]
> Do you have rehlds?
Yes, forgot to mention that. reHLDS can cause this issue? This is not a problem on YaPB.

**Comment by EfeDursun125** [comment]
some rehlds and regamedll versions can cause this, try latest ones

**Comment by PRinceBhai** [comment]
So reHLDS was a bit outdated, tried the version that was just released yesterday, It's still the same, they don't even connect automatically and when manually added they just stay on Spectators.

**Comment by Vladislav4KZ** [comment]
I'm seeing the same problem with the latest Steam build of Counter-Strike 1.6 without ReGameDLL, ReHLDS, Metamod, AMX Mod X.

Just vanilla game with "CS-EBOT" bots.

Counter-Strike 1.6 engine version (build):
```
Protocol version 48
Exe version 1.1.2.7/Stdio (cstrike)
Exe build: 19:53:27 Aug  3 2020 (8684)
```

E-BOT version:
```
------------------------------------------------
E-BOT 1.05
Build: 1.5.35821.6
------------------------------------------------
Builded by: EfeDursun125
URL: ebots-for-cs.blogspot.com 
Compiled: Sep 18 2022, 21:52:16 (UTC +08:00)
Meta-Interface Version: 5:13
------------------------------------------------
```
